.github/workflows/no-response.yml

```diff
@@ -0,0 +1,26 @@
+name: No Response
+
+## Closes issues marked with 'more information required' label after 7 days
+##    if original poster does not respond within that time
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
+        with:
+          token: ${{ secrets.SF_IDEE_BOT_GITHUB_TOKEN }}
+          daysUntilClose: 7
+          responseRequiredLabel: 'more information required'
+          closeComment: >
+            This issue has been automatically closed because there has been no response
+            to our request for more information from the original author. Currently, there
+            is not enough information provided for us to take action. Please reply and 
+            reopen this issue if you need additional assistance.
```

Comment:
Does it remove the label if it finds activity on the issue or no?

Reply:
Yep, this one does remove the label! It does have to be the original poster though.

Also one interesting [quirk](https://github.com/lee-dohm/no-response#issue_comment-event) that I don't think the app did: "Reopen the issue if it was closed by someone other than the original author of the issue" if the original author comments again. I think that's good though, because if we close it, but then the OG gets back to us, we probably want it to be reopened, and that'll save us a click.

Reply:
sweet. def sounds better